app/mailers/article_mailer.rb

```diff
@@ -13,6 +13,14 @@ def email_last_users_art(user_email, total_users, q_articles_per_user)
     @articles_per_user = q_articles_per_user
     mail subject:  "Check who's new @ the blog!",
          to:       user_email,
-         from:     Rails.application.secrets.credentials_user
+         from:     set_sender
+  end
+
+  def set_sender
+    if Rails.env.development?
+      Rails.application.secrets.credentials_user
```

Comment:
Rails.application.secrets uses the secrets.yml and there you can set the ENV['credentials_user']. So you don't need this if